cmd/krew/cmd/index.go

```diff
@@ -54,9 +55,31 @@ each configured index in table format.`,
 	},
 }
 
+var indexAddCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add a new index",
+	Long: `Configure a new index to install plugins from.
```

Comment:
See if you can name the positional args. 
Like NAME GIT_URL. 
Cobra probably has facilities to do that.